fn/result.go

```diff
@@ -62,15 +62,6 @@ func (r Result[T]) IsErr() bool {
 	return r.IsRight()
 }
 
-// Map applies an endomorphic function to the success value if it exists.
-//
-// Deprecated: Use MapOk instead.
-func (r Result[T]) Map(f func(T) T) Result[T] {
```

Comment:
Can be squashed with the commit where `Hush` is added.

Reply:
I think I'd rather keep the commits separate since this one is breaking.